src/utils/markdown.js

```diff
@@ -238,3 +240,113 @@ export function mdBtn(md) {
   );
 }
 
+export function myBackticks(md) {
```

Comment:
```suggestion
export function mdBackticks(md) {
```

right?

Reply:
where diid you get that code @idanizi-panw ? isn't it worth using existing plugins ?

Reply:
@Bars92 I took it from the `web-client`: https://code.pan.run/xsoar/web-client/-/blob/master/src/util/markdown.js#L325